Android/README.rst

```diff
@@ -0,0 +1,447 @@
+========================
+Python on Android README
+========================
+
+.. contents::
+
+This document provides a quick overview of the Python build system for Android
+on a **linux platform** [1]_.
```

Comment:
This documentation is uncomplete. You need "quick start" instructions to explain what is Android, what do I require on my local host, do I need a physical device, etc. This documentation mostly explains the implementation, not really how to use it.

Reply:
I'm trying to run Python on Android in an emulator using these recipes, but it's hard to understand how to use it. First, I downloaded NDK r16b but configure-android *immediately* fails because it *requires* NDK 14... This thing be very explicit in the doc, since the NDK is a big file (800 MB, so slow to download) and takes a lot of disk space (2.6 GB). It's not easy to install it.

Reply:
I will add a ``quick start`` section shortly.

> First, I downloaded NDK r16b but configure-android immediately fails because it requires NDK 14.

I understand your frustation. The README does say that ``android-ndk-r14 is required`` in a note and that note is referred to in each ``Requirements`` section where the NDK is required. I will add a warning about this in the ``quick start`` section unless you have a better idea.